ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlExceptionMapperTest.java

```diff
@@ -15,46 +15,49 @@
 
 package io.confluent.ksql.rest.server.resources;
 
+import static io.netty.handler.codec.http.HttpResponseStatus.INTERNAL_SERVER_ERROR;
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.CoreMatchers.instanceOf;
 import static org.hamcrest.CoreMatchers.sameInstance;
 import static org.hamcrest.MatcherAssert.assertThat;
 
+import io.confluent.ksql.api.server.OldApiExceptionMapper;
+import io.confluent.ksql.rest.EndpointResponse;
 import io.confluent.ksql.rest.Errors;
 import io.confluent.ksql.rest.entity.KsqlErrorMessage;
 import javax.ws.rs.WebApplicationException;
-import javax.ws.rs.core.Response;
 import org.junit.Test;
 
 public class KsqlExceptionMapperTest {
```

Comment:
nit: rename this to `OldApiExceptionMapperTest` to be consistent with the rename of `KsqlExceptionMapper` to `OldApiExceptionMapper`.

Reply:
ack